index.js

```diff
@@ -1,95 +1,137 @@
-// ***** Question 1 *****
+//***** Question 1 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
 
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+
+console.log("%cQuestion 1", "color: red") 
+
+let counter = 1
+
+counter = 2
+
+
+console.log("%c----------", "color: red") 
+
+console.log(counter) 
+// => 2
 
 
 // ***** Question 2 *****
 
 // *** Uncomment the lines below to test
 // *** after testing, comment the line causing the error back in 
 // *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
+console.log("%cQuestion 2", "color: red") 
+
+const name = 'Shelby'
 
-// console.log(name)
-// // => "Ian"
+console.log(name)
+// // // => "Ian"
 // name = "Not Ian" 
 // // => TypeError
-// console.log("%c----------", "color: red") 
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 3 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
+console.log("%cQuestion 3", "color: red") 
+
+function drinkWater(currentThirstLevel) {
+	console.log('Man I sure am thirsty');
+	currentThirstLevel -=1
+	console.log('Ahh that hits the spot');
+	return currentThirstLevel;
+
+}
 
-// console.log(drinkWater(12))
-// // => "Man I sure am thirsty"
-// // => "Ahh that hits the spot"
-// // => 11
-// console.log("%c----------", "color: red") 
+console.log(drinkWater(12))
+// => "Man I sure am thirsty"
+// => "Ahh that hits the spot"
+// => 11
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 4 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 4", "color: red") 
+console.log("%cQuestion 4", "color: red") 
+
+function sameSameButDifferent(num, maybeNum) {
+	if (num === maybeNum) {
+		return 'same same';
+	} else if (num === parseInt(maybeNum)) {
+	 return 'same same (but different)';
+	} else {
+		return 'different';
+	}
```

Comment:
Looks good! Nice use of `parseInt` here. Just a small note about indentation - here's how I'd format this:

```js
	if (num === maybeNum) {
		return 'same same';
	} else if (num === parseInt(maybeNum)) {
		return 'same same (but different)';
	} else {
		return 'different';
	}
```

The general idea is any time you create a block with { }, the code inside that block should be indented one level deeper.